tests/no_inst_fleet_unit_tests.py

```diff
@@ -0,0 +1,311 @@
+        if ((fuel_supply_point-demand_curve(1000,0,time_point))>(facility_tolerance*100)):
+            num = num + 1
+        else: 
+            num = num+0 
+
+    assert(num == 0)
+
+
+""" Test a-grow-1 
+        - a: only source facility 
+        - grow: growing demand of fuel commodity that drives deployment 
+        - 1: first test of a-const type 
+"""
+# Configuring it for this test instance 
+test_a_grow_1_temp = copy.deepcopy(TEMPLATE)
+test_a_grow_1_temp["simulation"].update({"region": {
+    "config": {"NullRegion": "\n      "},
+    "institution": {
+        "config": {
+            "NOInst": {
+                "calc_method": "arma", 
+                "commodities": {"val": ["fuel"]}, 
+                "demand_std_dev": "0.0", 
+                "growth_rate": "1.0", 
+                "initial_demand": "1000", 
+                "record": "1", 
+                "steps": "1"
+            }
+        },
+        "name": "source_inst"
+    },
+    "name": "SingleRegion"
+}
+}
+)
+
+# actual test part
+def test_a_grow_1():
+    output_file = 'test_a_grow_1_file.sqlite'
+    input_file = output_file.replace('.sqlite', '.json')
+    with open(input_file, 'w') as f:
+        json.dump(test_a_const_1_temp, f)
+    s = subprocess.check_output(['cyclus', '-o', output_file, input_file],
+                                universal_newlines=True, env=ENV)
+     check if ran successfully
+    assert("Cyclus run successful!" in s)
+
+    # getting the sqlite file
+    cur = get_cursor('test_a_grow_1_file.sqlite')
+
+    # check if supply of fuel is within facility_tolerance & catchup_tolerance
+    fuel_supply = cur.execute("select time, sum(value) from timeseriessupplyfuel group by time").fetchall()
+    
+    num = 0
+    for pt in range(catchup_tolerance,len(fuel_supply)):
+        fuel_supply_point = fuel_supply[pt][1]
+        time_point = fuel_supply[pt][0]
+        # if supply curve value is larger/smaller than demand curve by facility_tolerance 
+        # at any timestep (larger than catch up tolerance) the num counter will be larger 
+        # than 1 and the test will fail
+        if ((fuel_supply_point-demand_curve(1000,1,time_point))>(facility_tolerance*100)):
+            num = num + 1
+        else: 
+            num = num+0 
+
+    assert(num == 0)
+
+    """ Test a-grow-2 
+        - a: only source facility 
+        - grow: growing demand of fuel commodity that drives deployment 
+        - 2: 2nd test of a-const type (it has no initial demand)
+"""
+# Configuring it for this test instance 
+test_a_grow_2_temp = copy.deepcopy(TEMPLATE)
+test_a_grow_2_temp["simulation"].update({"region": {
+    "config": {"NullRegion": "\n      "},
+    "institution": {
+        "config": {
+            "NOInst": {
+                "calc_method": "arma", 
+                "commodities": {"val": ["fuel"]}, 
+                "demand_std_dev": "0.0", 
+                "growth_rate": "1.0", 
+                "initial_demand": "0", 
```

Comment:
Is this test relying on the grow of reactors to drive fuel demand, or just growing an arbitrary "fuel" commodity. The initital_demand = 0 seems like it will result in 0 growth?